fvm/src/engine/mod.rs

```diff
@@ -124,11 +124,11 @@ fn wasmtime_config(ec: &EngineConfig) -> anyhow::Result<wasmtime::Config> {
 
     let instance_count = ec.instance_pool_size();
     let instance_memory_maximum_size = ec.max_inst_memory_bytes;
-    if instance_memory_maximum_size % wasmtime_environ::WASM_PAGE_SIZE as u64 != 0 {
+    if instance_memory_maximum_size % wasmtime_environ::Memory::DEFAULT_PAGE_SIZE as u64 != 0 {
```

Comment:
This section should be pretty much identical to master.

Reply:
👍 confirmed, the only difference is the `c.macos_use_mach_ports` addition on master